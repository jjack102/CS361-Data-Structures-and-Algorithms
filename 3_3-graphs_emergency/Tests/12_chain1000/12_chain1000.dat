999 0 22
0 1 1 0
1 2 1 1
0 2 3 0
2 3 1 0
3 4 1 0
2 4 1 1
4 5 1 0
5 6 1 1
4 6 3 0
6 7 1 0
7 8 1 0
6 8 1 1
8 9 1 0
9 10 1 1
8 10 3 0
10 11 1 0
11 12 1 0
10 12 1 1
12 13 1 0
13 14 1 1
12 14 3 0
14 15 1 0
15 16 1 0
14 16 1 1
16 17 1 0
17 18 1 1
16 18 3 0
18 19 1 0
19 20 1 0
18 20 1 1
20 21 1 0
21 22 1 1
20 22 3 0
22 23 1 0
23 24 1 0
22 24 1 1
24 25 1 0
25 26 1 1
24 26 3 0
26 27 1 0
27 28 1 0
26 28 1 1
28 29 1 0
29 30 1 1
28 30 3 0
30 31 1 0
31 32 1 0
30 32 1 1
32 33 1 0
33 34 1 1
32 34 3 0
34 35 1 0
35 36 1 0
34 36 1 1
36 37 1 0
37 38 1 1
36 38 3 0
38 39 1 0
39 40 1 0
38 40 1 1
40 41 1 0
41 42 1 1
40 42 3 0
42 43 1 0
43 44 1 0
42 44 1 1
44 45 1 0
45 46 1 1
44 46 3 0
46 47 1 0
47 48 1 0
46 48 1 1
48 49 1 0
49 50 1 1
48 50 3 0
50 51 1 0
51 52 1 0
50 52 1 1
52 53 1 0
53 54 1 1
52 54 3 0
54 55 1 0
55 56 1 0
54 56 1 1
56 57 1 0
57 58 1 1
56 58 3 0
58 59 1 0
59 60 1 0
58 60 1 1
60 61 1 0
61 62 1 1
60 62 3 0
62 63 1 0
63 64 1 0
62 64 1 1
64 65 1 0
65 66 1 1
64 66 3 0
66 67 1 0
67 68 1 0
66 68 1 1
68 69 1 0
69 70 1 1
68 70 3 0
70 71 1 0
71 72 1 0
70 72 1 1
72 73 1 0
73 74 1 1
72 74 3 0
74 75 1 0
75 76 1 0
74 76 1 1
76 77 1 0
77 78 1 1
76 78 3 0
78 79 1 0
79 80 1 0
78 80 1 1
80 81 1 0
81 82 1 1
80 82 3 0
82 83 1 0
83 84 1 0
82 84 1 1
84 85 1 0
85 86 1 1
84 86 3 0
86 87 1 0
87 88 1 0
86 88 1 1
88 89 1 0
89 90 1 1
88 90 3 0
90 91 1 0
91 92 1 0
90 92 1 1
92 93 1 0
93 94 1 1
92 94 3 0
94 95 1 0
95 96 1 0
94 96 1 1
96 97 1 0
97 98 1 1
96 98 3 0
98 99 1 0
99 100 1 0
98 100 1 1
100 101 1 0
101 102 1 1
100 102 3 0
102 103 1 0
103 104 1 0
102 104 1 1
104 105 1 0
105 106 1 1
104 106 3 0
106 107 1 0
107 108 1 0
106 108 1 1
108 109 1 0
109 110 1 1
108 110 3 0
110 111 1 0
111 112 1 0
110 112 1 1
112 113 1 0
113 114 1 1
112 114 3 0
114 115 1 0
115 116 1 0
114 116 1 1
116 117 1 0
117 118 1 1
116 118 3 0
118 119 1 0
119 120 1 0
118 120 1 1
120 121 1 0
121 122 1 1
120 122 3 0
122 123 1 0
123 124 1 0
122 124 1 1
124 125 1 0
125 126 1 1
124 126 3 0
126 127 1 0
127 128 1 0
126 128 1 1
128 129 1 0
129 130 1 1
128 130 3 0
130 131 1 0
131 132 1 0
130 132 1 1
132 133 1 0
133 134 1 1
132 134 3 0
134 135 1 0
135 136 1 0
134 136 1 1
136 137 1 0
137 138 1 1
136 138 3 0
138 139 1 0
139 140 1 0
138 140 1 1
140 141 1 0
141 142 1 1
140 142 3 0
142 143 1 0
143 144 1 0
142 144 1 1
144 145 1 0
145 146 1 1
144 146 3 0
146 147 1 0
147 148 1 0
146 148 1 1
148 149 1 0
149 150 1 1
148 150 3 0
150 151 1 0
151 152 1 0
150 152 1 1
152 153 1 0
153 154 1 1
152 154 3 0
154 155 1 0
155 156 1 0
154 156 1 1
156 157 1 0
157 158 1 1
156 158 3 0
158 159 1 0
159 160 1 0
158 160 1 1
160 161 1 0
161 162 1 1
160 162 3 0
162 163 1 0
163 164 1 0
162 164 1 1
164 165 1 0
165 166 1 1
164 166 3 0
166 167 1 0
167 168 1 0
166 168 1 1
168 169 1 0
169 170 1 1
168 170 3 0
170 171 1 0
171 172 1 0
170 172 1 1
172 173 1 0
173 174 1 1
172 174 3 0
174 175 1 0
175 176 1 0
174 176 1 1
176 177 1 0
177 178 1 1
176 178 3 0
178 179 1 0
179 180 1 0
178 180 1 1
180 181 1 0
181 182 1 1
180 182 3 0
182 183 1 0
183 184 1 0
182 184 1 1
184 185 1 0
185 186 1 1
184 186 3 0
186 187 1 0
187 188 1 0
186 188 1 1
188 189 1 0
189 190 1 1
188 190 3 0
190 191 1 0
191 192 1 0
190 192 1 1
192 193 1 0
193 194 1 1
192 194 3 0
194 195 1 0
195 196 1 0
194 196 1 1
196 197 1 0
197 198 1 1
196 198 3 0
198 199 1 0
199 200 1 0
198 200 1 1
200 201 1 0
201 202 1 1
200 202 3 0
202 203 1 0
203 204 1 0
202 204 1 1
204 205 1 0
205 206 1 1
204 206 3 0
206 207 1 0
207 208 1 0
206 208 1 1
208 209 1 0
209 210 1 1
208 210 3 0
210 211 1 0
211 212 1 0
210 212 1 1
212 213 1 0
213 214 1 1
212 214 3 0
214 215 1 0
215 216 1 0
214 216 1 1
216 217 1 0
217 218 1 1
216 218 3 0
218 219 1 0
219 220 1 0
218 220 1 1
220 221 1 0
221 222 1 1
220 222 3 0
222 223 1 0
223 224 1 0
222 224 1 1
224 225 1 0
225 226 1 1
224 226 3 0
226 227 1 0
227 228 1 0
226 228 1 1
228 229 1 0
229 230 1 1
228 230 3 0
230 231 1 0
231 232 1 0
230 232 1 1
232 233 1 0
233 234 1 1
232 234 3 0
234 235 1 0
235 236 1 0
234 236 1 1
236 237 1 0
237 238 1 1
236 238 3 0
238 239 1 0
239 240 1 0
238 240 1 1
240 241 1 0
241 242 1 1
240 242 3 0
242 243 1 0
243 244 1 0
242 244 1 1
244 245 1 0
245 246 1 1
244 246 3 0
246 247 1 0
247 248 1 0
246 248 1 1
248 249 1 0
249 250 1 1
248 250 3 0
250 251 1 0
251 252 1 0
250 252 1 1
252 253 1 0
253 254 1 1
252 254 3 0
254 255 1 0
255 256 1 0
254 256 1 1
256 257 1 0
257 258 1 1
256 258 3 0
258 259 1 0
259 260 1 0
258 260 1 1
260 261 1 0
261 262 1 1
260 262 3 0
262 263 1 0
263 264 1 0
262 264 1 1
264 265 1 0
265 266 1 1
264 266 3 0
266 267 1 0
267 268 1 0
266 268 1 1
268 269 1 0
269 270 1 1
268 270 3 0
270 271 1 0
271 272 1 0
270 272 1 1
272 273 1 0
273 274 1 1
272 274 3 0
274 275 1 0
275 276 1 0
274 276 1 1
276 277 1 0
277 278 1 1
276 278 3 0
278 279 1 0
279 280 1 0
278 280 1 1
280 281 1 0
281 282 1 1
280 282 3 0
282 283 1 0
283 284 1 0
282 284 1 1
284 285 1 0
285 286 1 1
284 286 3 0
286 287 1 0
287 288 1 0
286 288 1 1
288 289 1 0
289 290 1 1
288 290 3 0
290 291 1 0
291 292 1 0
290 292 1 1
292 293 1 0
293 294 1 1
292 294 3 0
294 295 1 0
295 296 1 0
294 296 1 1
296 297 1 0
297 298 1 1
296 298 3 0
298 299 1 0
299 300 1 0
298 300 1 1
300 301 1 0
301 302 1 1
300 302 3 0
302 303 1 0
303 304 1 0
302 304 1 1
304 305 1 0
305 306 1 1
304 306 3 0
306 307 1 0
307 308 1 0
306 308 1 1
308 309 1 0
309 310 1 1
308 310 3 0
310 311 1 0
311 312 1 0
310 312 1 1
312 313 1 0
313 314 1 1
312 314 3 0
314 315 1 0
315 316 1 0
314 316 1 1
316 317 1 0
317 318 1 1
316 318 3 0
318 319 1 0
319 320 1 0
318 320 1 1
320 321 1 0
321 322 1 1
320 322 3 0
322 323 1 0
323 324 1 0
322 324 1 1
324 325 1 0
325 326 1 1
324 326 3 0
326 327 1 0
327 328 1 0
326 328 1 1
328 329 1 0
329 330 1 1
328 330 3 0
330 331 1 0
331 332 1 0
330 332 1 1
332 333 1 0
333 334 1 1
332 334 3 0
334 335 1 0
335 336 1 0
334 336 1 1
336 337 1 0
337 338 1 1
336 338 3 0
338 339 1 0
339 340 1 0
338 340 1 1
340 341 1 0
341 342 1 1
340 342 3 0
342 343 1 0
343 344 1 0
342 344 1 1
344 345 1 0
345 346 1 1
344 346 3 0
346 347 1 0
347 348 1 0
346 348 1 1
348 349 1 0
349 350 1 1
348 350 3 0
350 351 1 0
351 352 1 0
350 352 1 1
352 353 1 0
353 354 1 1
352 354 3 0
354 355 1 0
355 356 1 0
354 356 1 1
356 357 1 0
357 358 1 1
356 358 3 0
358 359 1 0
359 360 1 0
358 360 1 1
360 361 1 0
361 362 1 1
360 362 3 0
362 363 1 0
363 364 1 0
362 364 1 1
364 365 1 0
365 366 1 1
364 366 3 0
366 367 1 0
367 368 1 0
366 368 1 1
368 369 1 0
369 370 1 1
368 370 3 0
370 371 1 0
371 372 1 0
370 372 1 1
372 373 1 0
373 374 1 1
372 374 3 0
374 375 1 0
375 376 1 0
374 376 1 1
376 377 1 0
377 378 1 1
376 378 3 0
378 379 1 0
379 380 1 0
378 380 1 1
380 381 1 0
381 382 1 1
380 382 3 0
382 383 1 0
383 384 1 0
382 384 1 1
384 385 1 0
385 386 1 1
384 386 3 0
386 387 1 0
387 388 1 0
386 388 1 1
388 389 1 0
389 390 1 1
388 390 3 0
390 391 1 0
391 392 1 0
390 392 1 1
392 393 1 0
393 394 1 1
392 394 3 0
394 395 1 0
395 396 1 0
394 396 1 1
396 397 1 0
397 398 1 1
396 398 3 0
398 399 1 0
399 400 1 0
398 400 1 1
400 401 1 0
401 402 1 1
400 402 3 0
402 403 1 0
403 404 1 0
402 404 1 1
404 405 1 0
405 406 1 1
404 406 3 0
406 407 1 0
407 408 1 0
406 408 1 1
408 409 1 0
409 410 1 1
408 410 3 0
410 411 1 0
411 412 1 0
410 412 1 1
412 413 1 0
413 414 1 1
412 414 3 0
414 415 1 0
415 416 1 0
414 416 1 1
416 417 1 0
417 418 1 1
416 418 3 0
418 419 1 0
419 420 1 0
418 420 1 1
420 421 1 0
421 422 1 1
420 422 3 0
422 423 1 0
423 424 1 0
422 424 1 1
424 425 1 0
425 426 1 1
424 426 3 0
426 427 1 0
427 428 1 0
426 428 1 1
428 429 1 0
429 430 1 1
428 430 3 0
430 431 1 0
431 432 1 0
430 432 1 1
432 433 1 0
433 434 1 1
432 434 3 0
434 435 1 0
435 436 1 0
434 436 1 1
436 437 1 0
437 438 1 1
436 438 3 0
438 439 1 0
439 440 1 0
438 440 1 1
440 441 1 0
441 442 1 1
440 442 3 0
442 443 1 0
443 444 1 0
442 444 1 1
444 445 1 0
445 446 1 1
444 446 3 0
446 447 1 0
447 448 1 0
446 448 1 1
448 449 1 0
449 450 1 1
448 450 3 0
450 451 1 0
451 452 1 0
450 452 1 1
452 453 1 0
453 454 1 1
452 454 3 0
454 455 1 0
455 456 1 0
454 456 1 1
456 457 1 0
457 458 1 1
456 458 3 0
458 459 1 0
459 460 1 0
458 460 1 1
460 461 1 0
461 462 1 1
460 462 3 0
462 463 1 0
463 464 1 0
462 464 1 1
464 465 1 0
465 466 1 1
464 466 3 0
466 467 1 0
467 468 1 0
466 468 1 1
468 469 1 0
469 470 1 1
468 470 3 0
470 471 1 0
471 472 1 0
470 472 1 1
472 473 1 0
473 474 1 1
472 474 3 0
474 475 1 0
475 476 1 0
474 476 1 1
476 477 1 0
477 478 1 1
476 478 3 0
478 479 1 0
479 480 1 0
478 480 1 1
480 481 1 0
481 482 1 1
480 482 3 0
482 483 1 0
483 484 1 0
482 484 1 1
484 485 1 0
485 486 1 1
484 486 3 0
486 487 1 0
487 488 1 0
486 488 1 1
488 489 1 0
489 490 1 1
488 490 3 0
490 491 1 0
491 492 1 0
490 492 1 1
492 493 1 0
493 494 1 1
492 494 3 0
494 495 1 0
495 496 1 0
494 496 1 1
496 497 1 0
497 498 1 1
496 498 3 0
498 499 1 0
499 500 1 0
498 500 1 1
500 501 1 0
501 502 1 1
500 502 3 0
502 503 1 0
503 504 1 0
502 504 1 1
504 505 1 0
505 506 1 1
504 506 3 0
506 507 1 0
507 508 1 0
506 508 1 1
508 509 1 0
509 510 1 1
508 510 3 0
510 511 1 0
511 512 1 0
510 512 1 1
512 513 1 0
513 514 1 1
512 514 3 0
514 515 1 0
515 516 1 0
514 516 1 1
516 517 1 0
517 518 1 1
516 518 3 0
518 519 1 0
519 520 1 0
518 520 1 1
520 521 1 0
521 522 1 1
520 522 3 0
522 523 1 0
523 524 1 0
522 524 1 1
524 525 1 0
525 526 1 1
524 526 3 0
526 527 1 0
527 528 1 0
526 528 1 1
528 529 1 0
529 530 1 1
528 530 3 0
530 531 1 0
531 532 1 0
530 532 1 1
532 533 1 0
533 534 1 1
532 534 3 0
534 535 1 0
535 536 1 0
534 536 1 1
536 537 1 0
537 538 1 1
536 538 3 0
538 539 1 0
539 540 1 0
538 540 1 1
540 541 1 0
541 542 1 1
540 542 3 0
542 543 1 0
543 544 1 0
542 544 1 1
544 545 1 0
545 546 1 1
544 546 3 0
546 547 1 0
547 548 1 0
546 548 1 1
548 549 1 0
549 550 1 1
548 550 3 0
550 551 1 0
551 552 1 0
550 552 1 1
552 553 1 0
553 554 1 1
552 554 3 0
554 555 1 0
555 556 1 0
554 556 1 1
556 557 1 0
557 558 1 1
556 558 3 0
558 559 1 0
559 560 1 0
558 560 1 1
560 561 1 0
561 562 1 1
560 562 3 0
562 563 1 0
563 564 1 0
562 564 1 1
564 565 1 0
565 566 1 1
564 566 3 0
566 567 1 0
567 568 1 0
566 568 1 1
568 569 1 0
569 570 1 1
568 570 3 0
570 571 1 0
571 572 1 0
570 572 1 1
572 573 1 0
573 574 1 1
572 574 3 0
574 575 1 0
575 576 1 0
574 576 1 1
576 577 1 0
577 578 1 1
576 578 3 0
578 579 1 0
579 580 1 0
578 580 1 1
580 581 1 0
581 582 1 1
580 582 3 0
582 583 1 0
583 584 1 0
582 584 1 1
584 585 1 0
585 586 1 1
584 586 3 0
586 587 1 0
587 588 1 0
586 588 1 1
588 589 1 0
589 590 1 1
588 590 3 0
590 591 1 0
591 592 1 0
590 592 1 1
592 593 1 0
593 594 1 1
592 594 3 0
594 595 1 0
595 596 1 0
594 596 1 1
596 597 1 0
597 598 1 1
596 598 3 0
598 599 1 0
599 600 1 0
598 600 1 1
600 601 1 0
601 602 1 1
600 602 3 0
602 603 1 0
603 604 1 0
602 604 1 1
604 605 1 0
605 606 1 1
604 606 3 0
606 607 1 0
607 608 1 0
606 608 1 1
608 609 1 0
609 610 1 1
608 610 3 0
610 611 1 0
611 612 1 0
610 612 1 1
612 613 1 0
613 614 1 1
612 614 3 0
614 615 1 0
615 616 1 0
614 616 1 1
616 617 1 0
617 618 1 1
616 618 3 0
618 619 1 0
619 620 1 0
618 620 1 1
620 621 1 0
621 622 1 1
620 622 3 0
622 623 1 0
623 624 1 0
622 624 1 1
624 625 1 0
625 626 1 1
624 626 3 0
626 627 1 0
627 628 1 0
626 628 1 1
628 629 1 0
629 630 1 1
628 630 3 0
630 631 1 0
631 632 1 0
630 632 1 1
632 633 1 0
633 634 1 1
632 634 3 0
634 635 1 0
635 636 1 0
634 636 1 1
636 637 1 0
637 638 1 1
636 638 3 0
638 639 1 0
639 640 1 0
638 640 1 1
640 641 1 0
641 642 1 1
640 642 3 0
642 643 1 0
643 644 1 0
642 644 1 1
644 645 1 0
645 646 1 1
644 646 3 0
646 647 1 0
647 648 1 0
646 648 1 1
648 649 1 0
649 650 1 1
648 650 3 0
650 651 1 0
651 652 1 0
650 652 1 1
652 653 1 0
653 654 1 1
652 654 3 0
654 655 1 0
655 656 1 0
654 656 1 1
656 657 1 0
657 658 1 1
656 658 3 0
658 659 1 0
659 660 1 0
658 660 1 1
660 661 1 0
661 662 1 1
660 662 3 0
662 663 1 0
663 664 1 0
662 664 1 1
664 665 1 0
665 666 1 1
664 666 3 0
666 667 1 0
667 668 1 0
666 668 1 1
668 669 1 0
669 670 1 1
668 670 3 0
670 671 1 0
671 672 1 0
670 672 1 1
672 673 1 0
673 674 1 1
672 674 3 0
674 675 1 0
675 676 1 0
674 676 1 1
676 677 1 0
677 678 1 1
676 678 3 0
678 679 1 0
679 680 1 0
678 680 1 1
680 681 1 0
681 682 1 1
680 682 3 0
682 683 1 0
683 684 1 0
682 684 1 1
684 685 1 0
685 686 1 1
684 686 3 0
686 687 1 0
687 688 1 0
686 688 1 1
688 689 1 0
689 690 1 1
688 690 3 0
690 691 1 0
691 692 1 0
690 692 1 1
692 693 1 0
693 694 1 1
692 694 3 0
694 695 1 0
695 696 1 0
694 696 1 1
696 697 1 0
697 698 1 1
696 698 3 0
698 699 1 0
699 700 1 0
698 700 1 1
700 701 1 0
701 702 1 1
700 702 3 0
702 703 1 0
703 704 1 0
702 704 1 1
704 705 1 0
705 706 1 1
704 706 3 0
706 707 1 0
707 708 1 0
706 708 1 1
708 709 1 0
709 710 1 1
708 710 3 0
710 711 1 0
711 712 1 0
710 712 1 1
712 713 1 0
713 714 1 1
712 714 3 0
714 715 1 0
715 716 1 0
714 716 1 1
716 717 1 0
717 718 1 1
716 718 3 0
718 719 1 0
719 720 1 0
718 720 1 1
720 721 1 0
721 722 1 1
720 722 3 0
722 723 1 0
723 724 1 0
722 724 1 1
724 725 1 0
725 726 1 1
724 726 3 0
726 727 1 0
727 728 1 0
726 728 1 1
728 729 1 0
729 730 1 1
728 730 3 0
730 731 1 0
731 732 1 0
730 732 1 1
732 733 1 0
733 734 1 1
732 734 3 0
734 735 1 0
735 736 1 0
734 736 1 1
736 737 1 0
737 738 1 1
736 738 3 0
738 739 1 0
739 740 1 0
738 740 1 1
740 741 1 0
741 742 1 1
740 742 3 0
742 743 1 0
743 744 1 0
742 744 1 1
744 745 1 0
745 746 1 1
744 746 3 0
746 747 1 0
747 748 1 0
746 748 1 1
748 749 1 0
749 750 1 1
748 750 3 0
750 751 1 0
751 752 1 0
750 752 1 1
752 753 1 0
753 754 1 1
752 754 3 0
754 755 1 0
755 756 1 0
754 756 1 1
756 757 1 0
757 758 1 1
756 758 3 0
758 759 1 0
759 760 1 0
758 760 1 1
760 761 1 0
761 762 1 1
760 762 3 0
762 763 1 0
763 764 1 0
762 764 1 1
764 765 1 0
765 766 1 1
764 766 3 0
766 767 1 0
767 768 1 0
766 768 1 1
768 769 1 0
769 770 1 1
768 770 3 0
770 771 1 0
771 772 1 0
770 772 1 1
772 773 1 0
773 774 1 1
772 774 3 0
774 775 1 0
775 776 1 0
774 776 1 1
776 777 1 0
777 778 1 1
776 778 3 0
778 779 1 0
779 780 1 0
778 780 1 1
780 781 1 0
781 782 1 1
780 782 3 0
782 783 1 0
783 784 1 0
782 784 1 1
784 785 1 0
785 786 1 1
784 786 3 0
786 787 1 0
787 788 1 0
786 788 1 1
788 789 1 0
789 790 1 1
788 790 3 0
790 791 1 0
791 792 1 0
790 792 1 1
792 793 1 0
793 794 1 1
792 794 3 0
794 795 1 0
795 796 1 0
794 796 1 1
796 797 1 0
797 798 1 1
796 798 3 0
798 799 1 0
799 800 1 0
798 800 1 1
800 801 1 0
801 802 1 1
800 802 3 0
802 803 1 0
803 804 1 0
802 804 1 1
804 805 1 0
805 806 1 1
804 806 3 0
806 807 1 0
807 808 1 0
806 808 1 1
808 809 1 0
809 810 1 1
808 810 3 0
810 811 1 0
811 812 1 0
810 812 1 1
812 813 1 0
813 814 1 1
812 814 3 0
814 815 1 0
815 816 1 0
814 816 1 1
816 817 1 0
817 818 1 1
816 818 3 0
818 819 1 0
819 820 1 0
818 820 1 1
820 821 1 0
821 822 1 1
820 822 3 0
822 823 1 0
823 824 1 0
822 824 1 1
824 825 1 0
825 826 1 1
824 826 3 0
826 827 1 0
827 828 1 0
826 828 1 1
828 829 1 0
829 830 1 1
828 830 3 0
830 831 1 0
831 832 1 0
830 832 1 1
832 833 1 0
833 834 1 1
832 834 3 0
834 835 1 0
835 836 1 0
834 836 1 1
836 837 1 0
837 838 1 1
836 838 3 0
838 839 1 0
839 840 1 0
838 840 1 1
840 841 1 0
841 842 1 1
840 842 3 0
842 843 1 0
843 844 1 0
842 844 1 1
844 845 1 0
845 846 1 1
844 846 3 0
846 847 1 0
847 848 1 0
846 848 1 1
848 849 1 0
849 850 1 1
848 850 3 0
850 851 1 0
851 852 1 0
850 852 1 1
852 853 1 0
853 854 1 1
852 854 3 0
854 855 1 0
855 856 1 0
854 856 1 1
856 857 1 0
857 858 1 1
856 858 3 0
858 859 1 0
859 860 1 0
858 860 1 1
860 861 1 0
861 862 1 1
860 862 3 0
862 863 1 0
863 864 1 0
862 864 1 1
864 865 1 0
865 866 1 1
864 866 3 0
866 867 1 0
867 868 1 0
866 868 1 1
868 869 1 0
869 870 1 1
868 870 3 0
870 871 1 0
871 872 1 0
870 872 1 1
872 873 1 0
873 874 1 1
872 874 3 0
874 875 1 0
875 876 1 0
874 876 1 1
876 877 1 0
877 878 1 1
876 878 3 0
878 879 1 0
879 880 1 0
878 880 1 1
880 881 1 0
881 882 1 1
880 882 3 0
882 883 1 0
883 884 1 0
882 884 1 1
884 885 1 0
885 886 1 1
884 886 3 0
886 887 1 0
887 888 1 0
886 888 1 1
888 889 1 0
889 890 1 1
888 890 3 0
890 891 1 0
891 892 1 0
890 892 1 1
892 893 1 0
893 894 1 1
892 894 3 0
894 895 1 0
895 896 1 0
894 896 1 1
896 897 1 0
897 898 1 1
896 898 3 0
898 899 1 0
899 900 1 0
898 900 1 1
900 901 1 0
901 902 1 1
900 902 3 0
902 903 1 0
903 904 1 0
902 904 1 1
904 905 1 0
905 906 1 1
904 906 3 0
906 907 1 0
907 908 1 0
906 908 1 1
908 909 1 0
909 910 1 1
908 910 3 0
910 911 1 0
911 912 1 0
910 912 1 1
912 913 1 0
913 914 1 1
912 914 3 0
914 915 1 0
915 916 1 0
914 916 1 1
916 917 1 0
917 918 1 1
916 918 3 0
918 919 1 0
919 920 1 0
918 920 1 1
920 921 1 0
921 922 1 1
920 922 3 0
922 923 1 0
923 924 1 0
922 924 1 1
924 925 1 0
925 926 1 1
924 926 3 0
926 927 1 0
927 928 1 0
926 928 1 1
928 929 1 0
929 930 1 1
928 930 3 0
930 931 1 0
931 932 1 0
930 932 1 1
932 933 1 0
933 934 1 1
932 934 3 0
934 935 1 0
935 936 1 0
934 936 1 1
936 937 1 0
937 938 1 1
936 938 3 0
938 939 1 0
939 940 1 0
938 940 1 1
940 941 1 0
941 942 1 1
940 942 3 0
942 943 1 0
943 944 1 0
942 944 1 1
944 945 1 0
945 946 1 1
944 946 3 0
946 947 1 0
947 948 1 0
946 948 1 1
948 949 1 0
949 950 1 1
948 950 3 0
950 951 1 0
951 952 1 0
950 952 1 1
952 953 1 0
953 954 1 1
952 954 3 0
954 955 1 0
955 956 1 0
954 956 1 1
956 957 1 0
957 958 1 1
956 958 3 0
958 959 1 0
959 960 1 0
958 960 1 1
960 961 1 0
961 962 1 1
960 962 3 0
962 963 1 0
963 964 1 0
962 964 1 1
964 965 1 0
965 966 1 1
964 966 3 0
966 967 1 0
967 968 1 0
966 968 1 1
968 969 1 0
969 970 1 1
968 970 3 0
970 971 1 0
971 972 1 0
970 972 1 1
972 973 1 0
973 974 1 1
972 974 3 0
974 975 1 0
975 976 1 0
974 976 1 1
976 977 1 0
977 978 1 1
976 978 3 0
978 979 1 0
979 980 1 0
978 980 1 1
980 981 1 0
981 982 1 1
980 982 3 0
982 983 1 0
983 984 1 0
982 984 1 1
984 985 1 0
985 986 1 1
984 986 3 0
986 987 1 0
987 988 1 0
986 988 1 1
988 989 1 0
989 990 1 1
988 990 3 0
990 991 1 0
991 992 1 0
990 992 1 1
992 993 1 0
993 994 1 1
992 994 3 0
994 995 1 0
995 996 1 0
994 996 1 1
996 997 1 0
997 998 1 1
996 998 3 0
998 999 1 0
999 1000 1 0
998 1000 1 1
