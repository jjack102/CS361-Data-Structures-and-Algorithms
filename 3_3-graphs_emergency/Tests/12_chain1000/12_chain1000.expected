Fastest time for car at 999 to the emergency at 0 is 1227 minutes.
